doc/release/1.3.0-notes.rst

```diff
@@ -24,41 +24,572 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
 Highlights of this release
 --------------------------
 
--
+- Three new ``stats`` functions, a rewrite of ``pearsonr``, and an exact
+  computation of the Kolmogorov-Smirnov two-sample test
+- A new Cython API for bounded scalar-function root-finders in `scipy.optimize`
+- Substantial ``CSR`` and ``CSC`` sparse matrix indexing performance
+  improvements
+- added support for interpolation of rotations with continuous angular
+  rate and acceleration in ``RotationSpline``
+
 
```

Comment:
Do you want to mention anywhere the toolchain roadmap laying out plan for compatibility with Python/C/NumPy/...? Or is that information considered to be just for devs not users? 
I don't see an obvious existing section for it, so I'll just raise the question here.

Reply:
I think Ralf removed the milestone there, but I've added a concise note nonetheless.